code/modules/research/designs/medical_designs.dm

```diff
@@ -162,16 +172,6 @@
 	category = list("Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
 
```

Comment:
Honestly, i think Advanced Health analizers should be a diskette research rather than being gated off to admin only.